src/ShopBundle/mappers/basket/TPkgShopMapper_ArticleAddToBasket.class.php

```diff
@@ -41,7 +41,7 @@ public function Accept(IMapperVisitorRestricted $oVisitor, $bCachingEnabled, IMa
         $bRedirectToBasket = $oVisitor->GetSourceObject('bRedirectToBasket');
 
         $oVisitor->SetMappedValue('sAddToBasketLink', $oArticle->GetToBasketLink(false, $bRedirectToBasket, false, false, MTShopBasketCore::MSG_CONSUMER_NAME));
-        $aBasketParameters = $oArticle->GetToBasketLinkParameters($bRedirectToBasket, false, false, MTShopBasketCore::MSG_CONSUMER_NAME);
+        $aBasketParameters = $oArticle->GetToBasketLinkBasketParameters($bRedirectToBasket, false, false, MTShopBasketCore::MSG_CONSUMER_NAME);
```

Comment:
The private (and now unused) method getToBasketLinkOtherParameters() there could be deleted?

Reply:
It is not unused. `GetToBasketLinkParameters` stays public and it uses the this method.